extensions/guacamole-auth-cas/src/main/java/org/apache/guacamole/auth/cas/user/CASAuthenticatedUser.java

```diff
@@ -50,6 +51,11 @@
      */
     private Map<String, String> tokens;
 
+    /**
+     * The unique identifiers of all user groups which a user is a member of
```

Comment:
A period is missing here (JavaDoc requires that comments are terminated with punctuation - see the other JavaDoc comments in the vicinity for examples).

Reply:
Will fix, thank you.